test/core/derivatives/convolution_settings.py

```diff
@@ -276,3 +276,152 @@
         "id_prefix": "non-default-conv",
     },
 ]
+
+CONVOLUTION_GROUPS_SETTINGS = [
```

Comment:
I think you are overwriting the settings here, should it be `+=`?

Reply:
Maybe I also missunderstood something here. Where is `CONVOLUTION_GROUPS_SETTING` being used in the tests?

Reply:
This is another confusion I had. Did you want this to be like `LOSS_FAIL_SETTINGS`?

Reply:
Its not used anywhere as the settings here fail for `weights`. But maybe once I raise `NotImplementedError` correctly we wont need this